theme/gatsby-config.js

```diff
@@ -1,20 +1,23 @@
 module.exports = {
   siteMetadata: {
-    title: "Gatsby Theme Jam Example Submission",
+    title: "gatsby-theme-childrens-picture-book",
+    description:
+      "Make a Do-It-Yourself childrens picture book together with a child you know.",
```

Comment:
Spelling error "children's" not "childrens".

Reply:
theme/gatsby-config.js
`...
siteMetadata: {
    title: "gatsby-theme-children's-picture-book",
...`